plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -127,7 +127,20 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	}
 
 	trace.Step("Selecting host")
-	return g.selectHost(priorityList)
+	host, err := g.selectHost(priorityList)
+	if err != nil {
+		return "", err
+	}
+
+	if len(g.extenders) != 0 {
+		for _, extender := range g.extenders {
+			err := extender.Bind(pod, host)
```

Comment:
BTW, this is from the issue #41235:

Currently, the extender allows an external process to filter and prioritize nodes when scheduling a pod. But the process may also need to do preparation before binding Pod to host; for example, in kube-mesos, the k8sm-scheduler will ask Mesos to start kubelet before binding the Pod to host